src/util/sss_krb5.c

```diff
@@ -1330,3 +1330,22 @@ krb5_error_code sss_krb5_marshal_princ(krb5_principal princ,
     }
     return EOK;
 }
+
+krb5_error_code sss_krb5_init_context(krb5_context *context)
+{
+    krb5_error_code kerr;
+    const char *msg;
+
+    kerr = krb5_init_context(context);
+    if (kerr != 0) {
+        /* It is safe to call (sss_)krb5_init_context() with NULL as first
+         * argument. */
```

Comment:
LGTM,

I am just confused by this comment. Did you mean that as doc string comment or that it is safe
to pass `NULL` into `sss_krb5_get_error_message`?

Reply:
yes, that's a typo, if is sage to pass NULL to sss_krb5_get_error_message. I fixed it in the latest version.